### DISCUSSION TITLE
Reset statistics

### DISCUSSION THREAD

**Discussion by 1walkerit** [Arksine/moonraker#194] [Q&A]
Hi
is there any way how to reset statistics/history? May be I am blind but cannot find the way.
Thnak you for help.

**Comment by Arksine** [comment]
There is an [API to clear the history](https://moonraker.readthedocs.io/en/latest/web_api/#delete-job) if you are looking to implement it in your own client.  If you are using an existing client and want to clear the history you will want to check your client documentation.

The lifetime statistics cannot be reset without deleting the database.

**Reply by 1walkerit** [reply]
Thank you for tip and answer :-)
